apps/api/src/donations/donations.service.ts

```diff
@@ -583,7 +583,7 @@ export class DonationsService {
       //Donation exists, so updating with incoming donation without increasing vault amounts
       await this.prisma.donation.update({
         where: { extPaymentIntentId: donationDto.extPaymentIntentId },
-        data: donationDto,
+        data: { ...donationDto, updatedAt: existingDonation.updatedAt },
```

Comment:
Just for my curiosity... 
if the donation already exists - why do we even update it?

Reply:
I have no idea either thus decided to keep it for now, just in case of specific edge-case I can't think of. 

@BogoCvetkov is it safe to remove this update query?